copycat/api/src/main/java/org/apache/kafka/copycat/data/CopycatSchema.java

```diff
@@ -23,20 +23,37 @@
 import java.util.*;
 
 public class CopycatSchema implements Schema {
-    private static final Map<Type, Class<?>> SCHEMA_TYPE_CLASSES = new HashMap<>();
+    /**
+     * Maps Schema.Types to a list of Java classes that can be used to represent them.
+     */
+    private static final Map<Type, List<Class>> SCHEMA_TYPE_CLASSES = new HashMap<>();
```

Comment:
Maybe some explanation of when would a list with more than one item be useful?
The use-case is not completely obvious (You have schemas, they evolved, but you don't have a schema repository so you can't get the right version of the schema from an external system, IIRC).